manifests/07-operator.yaml

```diff
@@ -21,6 +21,7 @@ spec:
       priorityClassName: system-cluster-critical
       nodeSelector:
         node-role.kubernetes.io/master: ""
+        kubernetes.io/os: linux
```

Comment:
This is somewhat redundant - we only support RHCOS masters I believe.

Reply:
I vaguely recall a Jira story where we had to do something similar, i.e. make sure it doesn't run on windows, but I can't find the Jira story or analogous yaml in the repos I think I might have had to do this for.

Reply:
I see your point @adambkaplan. I will get rid of this.